packages/suite/src/middlewares/wallet/walletMiddleware.ts

```diff
@@ -96,6 +101,33 @@ const walletMiddleware =
             api.dispatch(coinmarketBuyActions.dispose());
         }
 
+        if (
+            prevState.router.pathname !== '/accounts/send' &&
+            nextRouter.pathname !== '/accounts/send' &&
```

Comment:
pathname will never be `/accounts/send` except localhost. It is `web/accounts/send` on production, `fix/satoshi-return-to-send/web/accounts/send` on this branch and so on. 

You could use router.route.name instead.

Reply:
fa39d3319c7953ea6e6d39c30bc33142d7481bb8